apps/web/src/views/Info/components/CurrencyLogo/index.tsx

```diff
@@ -20,20 +19,30 @@ export const CurrencyLogo: React.FC<
   React.PropsWithChildren<{
     address?: string
     token?: Token
+    currency?: Currency & {
```

Comment:
I don't think the currency is defined here? For info page we may not be able to get the currency/token instance. We probably need to add more utility functions like `getTokenLogoUrlsByAddress` to the widget. Like what we did here https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/widgets-internal/components/CurrencyLogo/utils.ts#L30-L40